tests/integration/usage_statistics/test_usage_statistics_messages.py

```diff
@@ -542,8 +592,8 @@ def test_generate_messages_with_defaults():
     ],
     "data_context.run_checkpoint": [
         {
-            "event_payload": {},
             "event": "data_context.run_checkpoint",
+            "event_payload": {},
```

Comment:
As noted in other comments - why are all the event_payloads blank for `data_context.run_checkpoint`?

Reply:
@anthonyburdi This is by specification/design -- this event is a no-payload event (indicating that the call occurred).